todo/src/App.js

```diff
@@ -44,27 +44,28 @@ export default function App() {
       </div>
 
       <div>
-        {list.map((item, index) => 
-          <div className={item.isChecked ? "Strike" : "Normal"} key={index}>
-            <Checkbox onChange={(e) => strikeThrough(e, index)} checked={item.isChecked}>
-              <Typography.Text
-                editable={{
-                  onChange: (newVal) => updateTask(newVal, index),
-                  maxLength: 100,
-                  autoSize: false,
-                }}
-              >
-                {item.text}
-              </Typography.Text>
-            </Checkbox>
-            <div className="delButton" key={index}>
-              <Button onClick={(e) => deleteTask(e, index)}>Delete</Button>
+        <div className="scrollBar">
+          {list.map((item, index) => 
+            <div className={item.isChecked ? "Strike" : "Normal"} key={index}>
+              <Checkbox onChange={(e) => strikeThrough(e, index)} checked={item.isChecked}>
+                <Typography.Text
+                  editable={{
+                    onChange: (newVal) => updateTask(newVal, index),
+                    maxLength: 100,
+                    autoSize: false,
+                  }}
+                >
+                  {item.text}
+                </Typography.Text>
+              </Checkbox>
+              <div className="delButton" key={index}>
+                <Button icon={<DeleteOutlined/>} danger onClick={(e) => deleteTask(e, index)}></Button>
```

Comment:
The danger property is supposed to make this red right?  Doesnt look like its doing that

Reply:
It makes the delete button red, which it is doing

Reply:
Yeah my bad I was looking at the wrong branch when I was testing this